### DISCUSSION TITLE
❓ Model Structure and Training

### DISCUSSION THREAD

**Discussion by hevinyu** [snakers4/silero-vad#27] [Q&A]
## ❓ Training details
Great, very exciting work. Unfortunately I couldn't find the details of the model structure and training. Very interested, could you share？

**Comment by snakers4** [comment]
Hi,

Model structure is public within the JIT files.

As for the training - we position our vad as a low level solution for end users to integrate into their apps (especially edge apps), not as framework / toolkit to build your own models.

If you have any feature requests - you are welcome, we aggregate the community feedback to prioritize next releases.

**Comment by snakers4** [comment]
Will convert this into a discussion
